genie-core/src/main/java/com/netflix/genie/core/jobs/JobDoneFile.java

```diff
@@ -17,24 +17,17 @@
  */
 package com.netflix.genie.core.jobs;
 
-import com.netflix.genie.common.dto.JobRequest;
+import lombok.Getter;
+import lombok.Setter;
 
 /**
- * Contains the logic to get all the details needed to run a job.
- * Resolves the criteria provided in the job request to construct the
- * JobExecutionEnvironment dto.
+ * Class that represents the structure of the genie.done file created when a job is done.
+ *
  * @author amsharma
+ * @since 3.0.0
  */
-public class JobEnvBuilder {
-
-    /**
-     * Method that accepts a jobRequest object and uses the criteria selected to build a
-     * jobExecutionEnvironmentBuilder object.
-     *
-     * @param jr The job request object
-     * @return jobExecutionEnvironment
-     */
-    public JobExecutionEnvironment build(final JobRequest jr) {
-        return null;
-    }
+@Getter
+@Setter
+public class JobDoneFile {
```

Comment:
Is there a case to be made to add a "reason" type field to this done file that the execution script can leverage? That way when we populate the job status message field of the job object in the database we can use this field to populate it if need be?